src/display/dom_utils.js

```diff
@@ -33,6 +33,36 @@ var createValidAbsoluteUrl = sharedUtil.createValidAbsoluteUrl;
 
 var DEFAULT_LINK_REL = 'noopener noreferrer nofollow';
 
+function DOMCanvasFactory() {}
```

Comment:
Sorry, i think i was little messed up with constructor function pattern. I will change it.